app/design/frontend/base/default/template/ebanx/form/boleto.phtml

```diff
@@ -1,13 +1,20 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-banking-ticket.png" width="40" /></div>';
 ?>
 
-<ul class="form-list ebanx-payment-method ebanx-br-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
+<?php if($showInlineIcon) {
```

Comment:
Yeah, I know I was the one who told you to do it like this, but I think @guilhermepiovesan was right. Maybe it's nicer to use `<?= $showInlineIcon ? $icon : '' ?>`